OgreMain/include/Ogre.i

```diff
@@ -768,6 +769,16 @@ SHARED_PTR(Material);
 %ignore Ogre::Mesh::getBoneAssignmentIterator;
 %template(PoseList) std::vector<Ogre::Pose*>;
 %template(SubMeshList) std::vector<Ogre::SubMesh*>;
+%include "typemaps.i"
+%define %standard_byref_params(TYPE)
+
+       %apply TYPE& INOUT { TYPE& };
+
+       %apply TYPE& OUTPUT { TYPE& outSourceCoordSet };
+       %apply TYPE& OUTPUT { TYPE& outIndex };
+
+%enddef
+%standard_byref_params(unsigned short)
```

Comment:
please use a more specifc %apply like here:
https://github.com/OGRECave/ogre/blob/e84d43eb5ec4b2017d02c18621d6aa0b69dea67f/OgreMain/include/Ogre.i#L367

that macro hides what you are actually doing here.